src/main/java/com/ericsson/eiffel/remrem/generate/App.java

```diff
@@ -7,26 +7,34 @@
 import org.springframework.context.ApplicationContext;
 import org.springframework.context.annotation.ComponentScan;
 
+import com.ericsson.eiffel.remrem.generate.cli.CLI;
+
 import java.util.Arrays;
 
 @SpringBootApplication
 @ComponentScan("com.ericsson.eiffel.remrem")
 public class App extends SpringBootServletInitializer {
 
     public static void main(String[] args) {
-        ApplicationContext ctx = SpringApplication.run(App.class, args);
-
-        System.out.println("Let's inspect active profiles:");
-        for (String envNames : ctx.getEnvironment().getActiveProfiles()) {
-            System.out.println(envNames);
-        }
-
-        System.out.println("Let's inspect the beans provided by Spring Boot:");
-
-        String[] beanNames = ctx.getBeanDefinitionNames();
-        Arrays.sort(beanNames);
-        for (String beanName : beanNames) {
-            System.out.println(beanName);
-        }
+    	
+    	CLI cli = new CLI();
+    	boolean startService = cli.parse(args);
```

Comment:
I think this is confusing. Add a comment about what is happening here. Something like "If there are command line options no service will be started". Change name of startService to needToStartService will also clarify.